src/pages/Inventory/AddItem.js

```diff
@@ -1,7 +1,119 @@
-import React from "react";
+import React, { Fragment, useState } from "react";
+import withWrapper from "common/withWrapper";
+import Navbar from "common/Navbar";
+import { useHistory } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import { useNotification } from "notification";
 
 const AddItem = () => {
-    return <div>I am add itme page</div>;
+    const [data, setData] = useState({
+        name: "",
+        price: "",
+        description: "",
+        date: new Date(),
+        id: Math.floor(Math.random() * 101 + 1),
+    });
+    const history = useHistory();
+    const { triggerNotification } = useNotification();
+
+    const AddItem = (e) => {
+        e.preventDefault();
+        // Adding to local storage
+        if (localStorage.getItem("inventory_data") == null) {
+            localStorage.setItem("inventory_data", "[]");
+        }
+        const inventoryData = JSON.parse(
```

Comment:
wrap JSON.parse with try catch if its fails for any reason UI will break

Reply:
Fixed